test/e2e/nodemaintenance_test.go

```diff
@@ -558,7 +618,7 @@ func checkFailureStatus(t *testing.T) {
 	}
 }
 
-func waitForDeployment(t *testing.T, namespace, name string, replicas int, retryInterval, timeout time.Duration) error {
+func waitForDeployment(t *testing.T, namespace, name string, retryInterval, timeout time.Duration) error {
```

Comment:
We never want deployments with more than a single copy of the Pod?

Reply:
It wasn't used, and I I didn't come up with a usecase... so I went the KISS way :shrug: